scripts/bash_aliases.sh

```diff
@@ -24,13 +24,13 @@ ros_env() {
 
 
 # Directory navigation
-alias mc="cd $CATKIN_DIR/src/mil_common"
+alias mc="cd \$CATKIN_DIR/src/mil_common"
```

Comment:
The change escapes the `$` character, which means the variable's value will be based on when the alias is used rather than when it is set.